Cadence/index.ts

```diff
@@ -55,10 +57,10 @@ export namespace Cadence {
 	/**
 	 * formula described here https://github.com/issuefab/app/issues/232
```

Comment:
Do we want to add the formula in the comments or are we fine with keeping it on github?

How is that going to work when/if we switch to github.com/proquse?